pkg/key/key.go

```diff
@@ -22,7 +25,7 @@ const (
 	// Release is the latest release version we use as base for conformance
 	// testing. This version changes with each new release so that we stay
 	// synchronized with the latest AWS release we publish.
-	Release = "11.5.0"
+	Release = "12.0.0"
```

Comment:
Latest release

Reply:
This should rather happen in master since I am not sure when this PR will get merged. Otherwise you will get merge conflicts if someone else decides to update this.